src/packages/@ncigdc/modern_components/ClinicalAnalysis/ClinicalVariableCard/utils/continuous.js

```diff
@@ -79,26 +78,26 @@ const parseContinuousValue = continuousValue =>
 
 export const parseContinuousKey = keyValue =>
   keyValue.split('-')
-    .map((val, idx, src) => (src[idx - 1] === '' ? `-${val}` : val))
-    .filter(val => val !== '')
-    .map(val => parseContinuousValue(val));
+    .map((val, idx, src) => (parseContinuousValue(src[idx - 1] === ''
```

Comment:
` .filter(val => val !== '')` is missing.

Reply:
DOH thank you